src/bundle/Resources/config/services/strategies.yaml

```
@@ -0,0 +1,19 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    _instanceof:
+        EzSystems\EzRecommendationClient\Strategy\Storage\GroupItemStrategyInterface:
+            tags:
+                - { name: ibexa.personalization.group_item.strategy }
+
+    EzSystems\EzRecommendationClient\Strategy\Storage\GroupByItemTypeAndLanguageStrategy: ~
+
+    EzSystems\EzRecommendationClient\Strategy\Storage\GroupItemStrategyDispatcher:
+        arguments:
+            $strategies: !tagged_iterator { tag: ibexa.personalization.group_item.strategy, default_index_method: getIndex }
```

As a note, I think I've used an expanded notation, so if you're nearing line length limit, you could use it instead:

```suggestion
            $strategies: !tagged_iterator 
                tag: ibexa.personalization.group_item.strategy
                default_index_method: getIndex
```

Just check that it works, cause I've typed it from the top of my head.

---

And you probably want to add `index: key` as well, just so that it's clear that this collection of services should be indexed using a specific tag attribute.